notification.go

```diff
@@ -171,6 +176,56 @@ func (n *notifier) GetServerInformation() (ServerInformation, error) {
 	return ret, nil
 }
 
+type NotificationClosedSignal struct {
+	Id uint32
+	Reason uint32
+}
+
+func (n *notifier) NotificationClosed(closed chan<- *NotificationClosedSignal) {
+	c := make(chan *dbus.Signal, len(closed))
+
+	go (func() {
+		for {
```

Comment:
Hmm, I think there needs to be a way for this go routine to exit, or it will leak?